roles/pulp_devel/tasks/rpm_signing_service.yml

```diff
@@ -0,0 +1,63 @@
+---
+# Setup the RPM signing service
+#
+# Maintain parity with:
+# https://github.com/pulp/pulp_rpm/blob/master/.travis/post_before_script.sh
+
+- name: Copy the signing service script to {{ __pulp_devel_rpm_sign_script_path }}
+  copy:
+    src: >-
+      {{ pulp_install_plugins_normalized['pulp-rpm'].source_dir | default (developer_user_home ~ '/devel/pulp_rpm')
+      }}/pulp_rpm/tests/functional/sign-metadata.sh
+    dest: "{{ __pulp_devel_rpm_sign_script_path }}"
+    remote_src: true
+    owner: "{{ pulp_user }}"
+    group: "{{ pulp_user }}"
+    mode: 0755
+  become: true
+  become_user: root
+
+- name: Import GPG key
+  shell: |
+    set -o pipefail
+    curl -L https://github.com/pulp/pulp-fixtures/raw/master/common/GPG-PRIVATE-KEY-pulp-qe | gpg --import
+  args:
+    executable: /bin/bash
+  # Retries in case github access issue.
+  retries: 3
+  delay: 3
+  register: result
+  until: result.rc == 0
+  changed_when: '"secret keys imported: 1" in result.stderr'
+  become: true
+  become_user: "{{ pulp_user }}"
+  when: not ansible_check_mode
```

Comment:
@fao89 This was the fix. It is documented [here](https://docs.ansible.com/ansible/latest/user_guide/playbooks_checkmode.html#id3).